dlp/snippets/deid_test.py

```diff
@@ -400,6 +400,23 @@ def test_deidentify_table_bucketing(capsys: pytest.CaptureFixture) -> None:
     assert 'string_value: "70:80"' in out
 
 
+def test_deidentify_table_primitive_bucketing(capsys: pytest.CaptureFixture) -> None:
+
+    deid.deidentify_table_primitive_bucketing(
+        GCLOUD_PROJECT,
+        TABLE_DATA["header"],
+        TABLE_DATA["rows"],
+        "happiness_score",
+        [0, 25, 75],
+        [25, 75, 100],
+        ["Low", "Medium", "High"],
```

Comment:
Can we rewrite this to accept the data in a form that is similar to bucket_config?
The three vectors are being used to assemble bucket_config. I am fine capturing this in the sample itself if needed.
The bucketing_upper_values, bucket_lowering_values and bucket_labels and how the example works is difficult to understand in this form.

Reply:
Yes, I also felt those arrays would be difficult to understand when writing this sample, but since table was inputted, other dependent fields should also be inputted. Passing values in the form of bucket_config would also be difficult for the user as user should know the protocol of bucket_config and code can be erroneous. Hence, I have hardcoded the table and related bucket configuration inside the function itself. Please check and let me know if this needs a change. Thank you.